assets/chat/js/chat.js

```diff
@@ -1357,6 +1362,38 @@ class Chat {
         this.applySettings();
     }
 
+    cmdHIDEEMOTE(parts, command) {
+        const hiddenEmotes = this.settings.get('hiddenemotes')
+        if (parts.length === 0) {
+            if (hiddenEmotes.length > 0) { MessageBuilder.info('Currently hidden emotes: ' + hiddenEmotes.join(',')).into(this); } else { MessageBuilder.info(`No hidden emotes`).into(this); }
+            return;
+        } 
+        if (!this.emoticons.has(parts[0])) {
+            MessageBuilder.info('Invalid emote. - /hideemote <emote>').into(this);
+            return;
+        }
+        const emote = parts[0]
+        const i = hiddenEmotes.indexOf(emote)
+        const emoteSpans = this.ui.find(`.chat-emote.chat-emote-${emote}`)
+        const emoteContainers = this.ui.find(`.generify-emote-${emote}`)
+        switch (command) {
```

Comment:
I think this switch statement needs formatting and I believe you can get rid of the default case here.